docs/dashboards-and-charts/netdata-charts.md

```diff
@@ -100,22 +100,27 @@ names:
 
 When you start interacting with a chart, you'll notice valuable information on the top bar:
 
-<img src="https://user-images.githubusercontent.com/70198089/236133832-fad45e65-5bd6-4fd1-8d68-33acf69fff5c.png" width="900"/>
+<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>
 
-The elements that you can find on this top bar are:
+The elements that you can find on this title bar are:
```

Comment:
```suggestion
Top bar elements:
```

We call it the top bar (the sentence above), why "title" bar here

Reply:
Let me check!

Reply:
I will replace top bar above, sounds better as "title bar"